packages/icons/icons/shadow-inset.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M1.64998 6.99998C1.64998 4.04525 4.04525 1.64998 6.99998 1.64998C9.9547 1.64998 12.35 4.04525 12.35 6.99998C12.35 9.9547 9.9547 12.35 6.99998 12.35C4.04525 12.35 1.64998 9.9547 1.64998 6.99998ZM6.99998 0.349976C3.32728 0.349976 0.349976 3.32728 0.349976 6.99998C0.349976 10.6727 3.32728 13.65 6.99998 13.65C10.6727 13.65 13.65 10.6727 13.65 6.99998C13.65 3.32728 10.6727 0.349976 6.99998 0.349976ZM4.31501 7.00001C4.31501 5.51713 5.51713 4.31501 7.00001 4.31501C8.4829 4.31501 9.68501 5.51713 9.68501 7.00001C9.68501 7.359 9.97603 7.65001 10.335 7.65001C10.694 7.65001 10.985 7.359 10.985 7.00001C10.985 4.79916 9.20087 3.01501 7.00001 3.01501C4.79916 3.01501 3.01501 4.79916 3.01501 7.00001C3.01501 7.359 3.30603 7.65001 3.66501 7.65001C4.024 7.65001 4.31501 7.359 4.31501 7.00001Z" fill="#11181C"/>
```

Comment:
See how every other svg is made. `fill` is removed from path and set as `currentColor`  at `svg` element level
Plz fix and regenerate